start.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 ROOTDIR=./php
 PORT=8100
-
 source ./sendgrid.env
+source  ./.env
 cd $ROOTDIR || exit
 
 #On iOS: this way built-in server supports ipv6 only
 # & (yarn install && yarn run start:dev)
 
 trap 'kill %1;' SIGINT
-(php -S 127.0.0.1:$PORT -c cli_server.ini) | tee php.log | sed -e 's/^/[PHP Server] /' &
+(php -S localhost:$PORT -c cli_server.ini) | tee php.log | sed -e 's/^/[PHP Server] /' &
```

Comment:
Unfortunately, I cant request 'localhost'. Can you use IP address, please?

Reply:
+